### DISCUSSION TITLE
Desktop Client on Self-Hosted

### DISCUSSION THREAD

**Discussion by lfaltran** [hoppscotch/hoppscotch#4109] [General]
Hello everyone, it's possible to use Hoppscotch Desktop Client with Self Hosted version? I think it's not, because there's any configuration file/option on desktop client to do that.

Maybe a proxy to "https://hoppscotch.io" redir  "https://hoppscotch.mycompany.com" should work?

Thank you in advice

**Comment by iAziz786** [comment]
We want the same feature and willing to contribute for it. Can anyone guide us through?

**Comment by CuriousCorrelation** [comment]
Hi, the latest desktop app has the ability to connect to self-hosted instances!

Feel free to download it from the [Hoppscotch](https://hoppscotch.com/download) website or the [releases section](https://github.com/hoppscotch/releases/releases).
